packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/lib/base/routers/router.dart

```diff
@@ -0,0 +1,9 @@
+import 'package:auto_route/auto_route.dart';
+import 'package:{{project_name}}/feature_counter/views/home_page.dart';
```

Comment:
Change this import to a relative one

Reply:
@DDavidPrime, git says that the import is still absolute, could you please double-check this. Thanks